package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "babel-plugin-inline-react-svg",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Please revert this; the version number should never change in a PR, and “adding support” for something is always semver-minor anyways.
```suggestion
  "version": "1.0.0",
```